### PR TITLE
Conversions: now supports speed!

### PR DESCRIPTION
**Pull request by mintsoft**
Currently the supported units are:
* m/s
* ft/s
* mph
* km/h
* knots

Will hopefully fix: https://github.com/duckduckgo/zeroclickinfo-goodies/issues/2729
and partially address: https://github.com/duckduckgo/zeroclickinfo-goodies/issues/689

Suggestions for new units welcome, however I don't want to go too berzerk on this as the number of combinations is huge and I'd rather solve it for everything.

cc @duckduckgo/community-leaders @moollaza @tagawa 

--------
https://duck.co/ia/view/conversions

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/conversions/ratios.yml [hunk 1]
```diff
@@ -1277,3 +1277,55 @@ aliases:
...
+---
+aliases:
+  - meters per second
+  - metres per second
+  - meter per second
+  - metre per second
+  - meter/second
+  - metre/second
+  - meters/second
+  - metres/second
```
Comment (GuiltyDolphin) [inline share/goodie/conversions/ratios.yml]: Can we add `mps`?

Comment (mintsoft) [inline share/goodie/conversions/ratios.yml]: Added

File: share/goodie/conversions/ratios.yml [hunk 2]
```diff
@@ -1277,3 +1277,56 @@ aliases:
 factor: 1.03397576569129e-25
 type: digital
 unit: yobibyte
+---
+aliases:
+  - mi/h
+  - miles per hour
```
Comment (GuiltyDolphin) [inline share/goodie/conversions/ratios.yml]: Might want to add `mile per hour`. (`1 mile per hour in kmph`...)

File: share/goodie/conversions/ratios.yml [hunk 3]
```diff
@@ -1277,3 +1277,56 @@ aliases:
...
+  - ft/sec
+  - ft/second
+factor: 3.281
+type: speed
+unit: ft/s
+---
+aliases:
+  - kilometer per hour
+  - kilometre per hour
+  - kph
```
Comment (GuiltyDolphin) [inline share/goodie/conversions/ratios.yml]: Typo? Or at least we should support `kmph`.

File: share/goodie/conversions/ratios.yml (hunk 2 above)
Comment (mintsoft) [inline share/goodie/conversions/ratios.yml]: sorted

File: share/goodie/conversions/ratios.yml (hunk 3 above)
Comment (mintsoft) [inline share/goodie/conversions/ratios.yml]: added

File: share/goodie/conversions/ratios.yml [hunk 4]
```diff
@@ -1277,3 +1277,58 @@ aliases:
...
+  - meter per second
+  - metre per second
+  - meter/second
+  - metre/second
+  - meters/second
+  - metres/second
+  - mps
+factor: 1
+type: speed
+unit: m/s
```
Comment (GuiltyDolphin) [inline share/goodie/conversions/ratios.yml]: If I do `1 mps in mph` I get `1 m/ = 2.237 mph`; maybe a plural/singular exception needed?

File: t/Conversions.t [hunk 5]
```diff
@@ -2529,6 +2529,42 @@ ddg_goodie_test(
...
+            markup_input => '10',
+            raw_input => '10',
+            from_unit => 'm/s',
+            styled_output => '32.810',
+            raw_answer => '32.810',
+            to_unit => 'ft/s',
+            physical_quantity => 'speed'
+        })
+    ),
+    '10 km/h to mph' => test_zci(
```
Comment (GuiltyDolphin) [inline t/Conversions.t]: After mentioning that above - can we get a couple of 'singular' tests? E.g., `ft/s in kmph`.

Comment (mintsoft) [inline t/Conversions.t]: Added

Comment (GuiltyDolphin) [inline t/Conversions.t]: Could we get specifically `ft/s to m/s`?

File: share/goodie/conversions/ratios.yml (hunk 4 above)
Comment (mintsoft) [inline share/goodie/conversions/ratios.yml]: Added

File: t/Conversions.t (hunk 5 above)
Comment (mintsoft) [inline t/Conversions.t]: Was committing that as you responded

Comment (GuiltyDolphin) [inline t/Conversions.t]: Heheh :wink:

**Comment by GuiltyDolphin** [comment]
Righty, thanks @mintsoft!

Lets get this on Beta @moollaza, I'm sure if there's any pressing issues they'll be picked up pretty quickly.

**Comment by GuiltyDolphin** [comment]
@MrChrisW @mintsoft Is this triggering for you on Beta? I've tried `1 m/s in km/h` and `kmph in mph` (and others), it doesn't appear to be triggering for me.

**Comment by mintsoft** [comment]
@GuiltyDolphin I think Conversions is totalled on beta for some reason; no queries seem to work:
https://beta.duckduckgo.com/?q=10+m+in+inches&ia=iatesting

**Comment by moollaza** [comment]
@mintsoft @GuiltyDolphin deployed to Beta. It's working again ;)

**Comment by moollaza** [comment]
Thanks! This LGTM :+1:

Seems like our numbers are a few decimal places off from Google FWIW:
50m_s_to_mph_-_google_search_and_50knots_to_mph_at_duckduckgo
50m_s_to_mph_-_google_search_and_50m_s_to_mph_at_duckduckgo

File: share/goodie/conversions/ratios.yml [hunk 6]
```diff
@@ -1277,3 +1277,58 @@ aliases:
 factor: 1.03397576569129e-25
 type: digital
 unit: yobibyte
+---
+aliases:
+  - mi/h
+  - miles per hour
+  - mile per hour
```
Comment (moollaza) [inline share/goodie/conversions/ratios.yml]: autocomplete has queries for `miles/hour`

https://beta.duckduckgo.com/?q=1+knot+in+miles%2Fhour&t=h&ia=iatesting

Comment (moollaza) [inline share/goodie/conversions/ratios.yml]: 1_knot_in_miles_hour_at_duckduckgo

**Comment by GuiltyDolphin** [comment]
@moollaza, @mintsoft: I checked it out, @moollaza is right - we're a bit inaccurate (50mph ~ 57.53895 knots) - we might want to check the precision we go to in the units and make sure we never exceed that in displaying results.

**Comment by mintsoft** [comment]
@GuiltyDolphin It should be as simple as just increasing the accuracy in the yaml; I think I only did like 4DP on the ratios

**Comment by GuiltyDolphin** [comment]
@mintsoft M'kay - is that in-progress in any of your PRs, or do you want to do a separate PR for that?

**Pull request merged**